src/layouts/default.vue

```diff
@@ -20,6 +21,11 @@
         :class="{ 'border-dark-charcoal-20 border-s': isSidebarVisible }"
       />
     </main>
+
+    <VFooter
```

Comment:
This will need to add the sidebar padding as the `main` does, otherwise the sidebar covers a part of the footer:

<img width="842" alt="Screen Shot 2022-09-19 at 9 05 45 PM" src="https://user-images.githubusercontent.com/15233243/191084372-39b1a6fb-e283-4555-80ec-7468dc062cf0.png">

Reply:
If the footer is shrunk in width to accommodate the sidebar, the responsiveness of the footer will break down. It will be using a wider style (based on the viewport width) in a narrower area and will look pretty bad.

Reply:
@dhruvkb I what Olga is suggesting might be the only viable solution. I don't think we can acceptably cover the footer with the sidebar. I tried some approaches: to make the sidebar height shorter so it ends before the footer; and to give the footer a higher z-index and add padding to the bottom of the filter sidebar, but neither worked well.

Reply:
> It will be using a wider style (based on the viewport width) in a narrower area

I think we'll have to take a JS-based approach to setting the right footer style at the right time, sadly.

Reply:
Implemented in 93339e9.